doc/classes/OS.xml

```diff
@@ -275,28 +284,32 @@
 			<return type="int">
 			</return>
 			<description>
-				Returns the amount of battery left in the device as a percentage.
+				Returns the amount of battery left in the device as a percentage. Returns [code]-1[/code] if power state is unknown.
+				[b]Note:[/b] This method is implemented on Android, iOS, Linux, macOS, Windows and UWP.
```

Comment:
All power battery functions are only functional on Linux (or should be). They are not properly implemented on other platforms. 
As an example, the method that does the job on iphone is here: https://github.com/godotengine/godot/blob/master/platform/iphone/power_iphone.cpp#L34

Reply:
Indeed iOS and UWP implementations are empty.

- Android have some code, but doesn't work on my device
- macOS have some code, on the device without battery returns POWERSTATE_NO_BATTERY)
- Windows also do call legit [GetSystemPowerStatus](https://docs.microsoft.com/en-us/windows/win32/api/winbase/nf-winbase-getsystempowerstatus) API, and works OK (both % and status)

Reply:
I am a bit responsible of this situation TBH :$
Problem is, I don't have the testing platforms myself to get this to work.